### PR TITLE
API refactored for Admin

### PR DESCRIPTION
**Pull request by codemutatio**
refactored the location and campaign API for different Admin usage

### CODE REVIEW

File: functions/src/controllers/campaign.js [hunk 1]
```diff
@@ -16,12 +18,63 @@ const Campaign = {
...
+            tryCatchError(res, error);
+        }
+    },
+
+    async campaignApproved(req, res) {
+        try {
+            const document = db.collection('campaigns').doc(req.params.id);
+            if (!document) validationError(res, errors);
+            req.body.status = approved;
+            req.body.staus = live;
```
Comment (judeokafor) [inline functions/src/controllers/campaign.js]: There is a mix up here i guess

Comment (judeokafor) [reply functions/src/controllers/campaign.js]: It should only be approved until its been sent to pipul tv before we update status to live

**Review by judeokafor — APPROVED** [review]
Cool. Minor changes

File: functions/src/controllers/campaign.js (hunk 1 above)
Comment (enyeInc) [reply functions/src/controllers/campaign.js]: fix this and then the code is good to merge

**Pull request merged**